#//////////////////////////////////////////////////////////////////////
#//////////////////////////////////////////////////////////////////////
# ------------------ point_cloud_compare Parameters -------------------
#//////////////////////////////////////////////////////////////////////
#//////////////////////////////////////////////////////////////////////

# File Locations and Names
groundTruthLocation: /home/karrer/FinalDataSet/Aligning
groundTruthName: Desk
reconstructionLocation:  /home/karrer/IROS2016/Evaluation/Okvis/Picoflexx/Desk/desk1
reconstructionName: desk1_1

# Save the point cloud
saveName: desk_1_1_diff

# Subsampling grid-size (no subsampling if set to 0.0)
subsampleGroundTruth: 0.0
subsampleReconstruction: 0.0

# Error Resolution (maximal Error to be distinguished)
largestError: 0.025

# ICP Parameters
maxIterations: 15
maxDistance: 0.1

# Filter Parameters
filterRadius: 0.02
filterNeighbours: 15

# Final processing
reorientatePC: false
